components/create_comment/create_comment.tsx

```diff
@@ -312,8 +345,8 @@ class CreateComment extends React.PureComponent {
         }
     }
 
-    componentDidUpdate(prevProps, prevState) {
-        if (prevState.draft.uploadsInProgress.length < this.state.draft.uploadsInProgress.length && this.props.scrollToBottom) {
+    componentDidUpdate(prevProps: Props, prevState: State) {
+        if (prevState.draft!.uploadsInProgress.length < this.state.draft!.uploadsInProgress.length && this.props.scrollToBottom) {
```

Comment:
Same with draft

Reply:
It will be undefined the short time between the constructor is called, and the getDerivedStateFromProps is called. We cannot set it during the constructor because getDerivedStateFromProps is relying on it being undefined.